moderator/settings.py

```diff
@@ -248,3 +248,24 @@ def _username_algo(email):
 
 # Enable dev login
 ENABLE_DEV_LOGIN = config("ENABLE_DEV_LOGIN", default=False, cast=bool)
+FROM_NOREPLY = config(
+    "FROM_NOREPLY",
+    default="No Reply - Mozilla Moderator <no-reply@moderator-mozilla.org",
```

Comment:
so the domain of the sender email address needs to match the domain the application is running in - e.g. no-reply@moderator-stage.itsre-apps.mozit.cloud for staging and no-reply@moderator.mozilla.org for production. Anything else, SES won't send it (it would be an unvalidated domain).

There might be a clever way to use one of the existing envvars for this, but they're full URLs and not just domains atm. Or, you may have this already covered through config work.